setup.cfg

```diff
@@ -1,5 +1,6 @@
 [bumpversion]
-current_version = "1.0"
+current_version = 1.0.0
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(-(?P<release>.*)-(?P<build>\d+))?
```

Comment:
What is this for?

Reply:
As far as I can tell, this is only useful if you are planing to have an advanced release versioning scheme, in which case it is useful to define [bumpversion:part:release] and [bumpversion:part:build] sections in the `setup.cfg`. I really suggest you keep this simple until you get familar with versioning.

Reply:
Parse is needed to get the version changed and the version parsed correctly